static/docs/use-cases/shared-development-server.md

```diff
@@ -69,10 +69,10 @@ $ git commit -m "cleanup raw data"
 $ git push
 ```
 
-Your colleagues can pull the project, and have both `raw` and `clean` data files
-instantly appear in their workspace without moving anything manually. After
-this, they could decide to continue building this pipeline and process the
-cleaned up data:
+Your colleagues can [checkout](/doc/command-reference/checkout) the project
+data, and have both `raw` and `clean` data files instantly appear in their
```

Comment:
BTW, "instantly appear"? Actually DVC has been becoming slower recently. Should we use the term "quickly" instead on "instantly" in most docs? (except perhaps marketing-oriented pages) or is this slowing temporary @shcheklein ? (In any case it will never be truly instantaneous.)

Reply:
in case of shared dev server they supposed to use some kind of links (should we review and update the page?). In this case it should more or less instant. I agree though that `quickly` is a safer choice though. No copying is happening. @efiop could you chime in please? Is it instant now if it's a shared cache?

Reply:
I thought about it and adding a note about file links in this document didn't seem necessary. It doesn't change anything, it would just be a disclaimer for the term "instant" which seems like we want to change anyway, right?

BTW I'm on a Mac so it should be using reflinks. Still all `dvc` commands take several seconds, even `dvc version`. There's some sort of noticeable overhead both in my pip-installed `dvc` and on the actual core repo in dev mode.

Reply:
p.s. updated wording in 4c42677 so feel free to resolve this conversation if there's nothing more to add 🙂

Reply:
@jorgeorpinel in case of data if time is a few seconds we can def consider that as instantaneous :)

Reply:
Idk. "instantaneous" seems misleading to me.

![image](https://user-images.githubusercontent.com/1477535/66250417-6f0b1100-e710-11e9-8f61-a4cbcedbbabe.png)

And `dvc version` takes 13 seconds on my Mac BTW ):  But that's a separate problem I guess will be addressed when a performance profiler system is established (:

Reply:
`dvc version` 13 seconds is a bug!!! Please, create a ticket at `iterative/dvc` to investigate this.

```
$ time dvc versions

real    0m0.140s
user    0m0.117s
sys     0m0.024s
```

on https://katacoda.com/dvc/courses/basics/data

Reply:
Opened https://github.com/iterative/dvc/issues/2582